docs/marketplace/fastlane/testing-distribution/index.md

```diff
@@ -58,6 +63,10 @@ fastlane add_plugin appcircle_testing_distribution
 - `personalAPIToken`: The Appcircle Personal API token is used to authenticate and secure access to Appcircle services. Add this token to your credentials to enable its use in your pipeline and ensure authorized actions within the platform.
 - `profileName`: Specifies the profile that will be used for uploading the app.
 - `createProfileIfNotExists`: Ensures that a user profile is automatically created if it does not already exist; if the profile name already exists, the app will be uploaded to that existing profile instead.
+- `profileCreationSettings`: If `createProfileIfNotExists` is `true` and a new profile being created, the profile will be configured with this settings.
+   - `authType`: Authentication type of the profile. `0`: None, `1`: Static Username and Password, `2`: LDAP, `3`: SSO.
```

Comment:
Option names should be the same as what we see in the UI. For example, instead of using "LDAP", we should use "LDAP Login".
![Screenshot 2024-12-11 at 16 07 49](https://github.com/user-attachments/assets/0ded71b5-c5d3-488f-9ad2-511a6472bd2c)

Note: Please check the option naming also for the SSO login.